BUILD.gn

```diff
@@ -349,6 +349,9 @@ action("create_symbols_dist") {
 
   inputs = get_target_outputs(":brave_license_dist_resources")
   dir_inputs = [ "$brave_product_name.breakpad.syms" ]
+  if (brave_debug_symbols_level == 2) {
```

Comment:
I'm not following why we'd generate and include breakpad.syms, but not pdb.syms? Either we want symbols or we don't.

Reply:
Agreed.
But right now we are uploading symbols for PRs build. So if I disabled generating `breakpad.syms` now it will breaks CI.
After we modify CI code I will be happy to set it `0` by default (and add some more ifs to not create `$brave_product_name.breakpad.syms`)